src/ppxlib.ml

```diff
@@ -1,5 +1,76 @@
 (** Standard library for ppx rewriters *)
 
+(** [ppxlib] is supposed to be opened globally in your PPX source files.
+
+    Opening it will bring several modules in scope, that are useful to have when
+    writing a rewriter:
+
+    - The "core" [ppxlib] modules, such as modules to help manipulate the AST
+      ({!Ast_builder}, {!Ast_pattern}), and a few functions.
```

Comment:
```suggestion
    - The main [ppxlib] modules, such as modules to help manipulate the AST
      ({!Ast_builder}, {!Ast_pattern}), and a few functions.
```

Just to avoid any wrong association with the Jane Street `core` library. Quite a few people have a strong opinion about avoiding `base` and `core` (and some people associate Ppxlib with them, since we used to have them as dependencies, but we've dropped them some time ago).